src/actions/actions.py

```diff
@@ -77,6 +77,7 @@ class CallbackAction:
     "extract": "extractVariable",
     "float": "insertEmptyLineAfter",
     "fold": "foldRegion",
+    "give": "removeSelection",
```

Comment:
I worry that "remove" is ambiguous here, because could imply removing the text of the selection. Maybe "deselect"?

Reply:
Done